.github/workflows/starlingmonkey.yml

```diff
@@ -107,7 +107,7 @@ jobs:
 
     - name: Run tests
       timeout-minutes: 20
-      run: node ./tests/wpt-harness/run-wpt.mjs --starlingmonkey -vv
+      run: node ./tests/wpt-harness/run-wpt.mjs -vv
```

Comment:
Was removing the `--starlingmonkey` flag intentional?

Reply:
Yes, I originally was going to use this flag to have separate expectations for StarlingMonkey and js-compute-runtime, but was able to get them to match up in the end.